### PR TITLE
Fixed: ERC20 token gas value issue

### PR DESCRIPTION
**Pull request by iosdevashok**
IMAGE 2023-08-10 17:43:10

### CODE REVIEW

**Review by madtocc — CHANGES_REQUESTED** [review]
Gas symbol is not correct: 
It should be the primary currency of the network (ETH, BNB, etc... not the erc20 token)

Gas used is not correct:
To calculate the total cost of the transaction in wei, you simply multiply the gasUsed by the effectiveGasPrice. Then, to represent that value in ETH (or the primary currency of whatever Ethereum-based network you're on), you would divide by 10^18.

Here's a breakdown:
Convert the hexadecimal gasUsed and effectiveGasPrice to decimal:

gasUsed: 0x1e406 = 124422 (in decimal)
effectiveGasPrice: 0xb2d05e00 = 3000599552 (in decimal)
Calculate the total cost in wei:

total_cost_wei = gasUsed * effectiveGasPrice
total_cost_wei = 124422 * 3000599552 = 373383540354784
Convert the total cost to ETH:

total_cost_eth = total_cost_wei / 10^18
total_cost_eth = 373383540354784 / 10^18 = 0.000373383540354784 ETH
So, the transaction cost approximately 0.000373383540354784 ETH in gas.

**Comment by iosdevashok** [comment]
https://github.com/pellartech/carbon-ios/assets/102407838/d5ac7947-7850-4d72-bb8e-bb9d0081b79d

File: 1 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Merge branch 'dev' into fix/erc20-gas-prize
```
Comment (madtocc) [inline 1:1]: ???

**Review by madtocc — CHANGES_REQUESTED** [review]
file not needed...

**Review by madtocc — APPROVED** [review]

**Pull request merged**